mne/time_frequency/psd.py

```diff
@@ -181,16 +202,29 @@ def psd_welch(inst, fmin=0, fmax=np.inf, tmin=None, tmax=None, n_fft=256,
         Evoked object. Defaults to True.
 
         .. versionadded:: 0.15.0
+    combine : str | float | callable | None
+        The type of reduction to perform on welch windows. If string it can be
+        'mean' or 'median'. If float it is understood as the proportion of
+        values to trim before performing mean (has to be > 0 and < 0.5) ie.
+        trimmed-mean. If function it has to perform the reduction along last
+        dimension. If None, no reduction is performed and psd's for individual
+        windows are returned. In such case the output PSDs have one more
+        dimension than the input array with windows at -3 dimension.
+
+        .. versionadded:: 0.15.0
     verbose : bool, str, int, or None
         If not None, override default verbose level (see :func:`mne.verbose`
         and :ref:`Logging documentation <tut_logging>` for more).
 
     Returns
     -------
     psds : ndarray, shape (..., n_freqs)
-        The power spectral densities. If input is of type Raw,
-        then psds will be shape (n_channels, n_freqs), if input is type Epochs
-        then psds will be shape (n_epochs, n_channels, n_freqs).
+        The power spectral densities. If input is of type Raw, returned psds
+        are shaped (n_channels, n_freqs), if input is type Epochs returned psds
+        are shaped (n_epochs, n_channels, n_freqs). However if combine is None
+        then if input is of type Raw, returned psds will be shaped (n_windows,
+        n_channels, n_freqs), if input is type Epochs then psds will be shaped
+        (n_epochs, n_windows, n_channels, n_freqs).
```

Comment:
I'm sorry if this was already discussed, but for me it is a bit of a surprise to get different shape compared to ``psd_array_welch``.